wasm.md

```diff
@@ -468,15 +480,128 @@ Wrapping cuts of the 32 most significant bits of an `i64` value.
 Extension turns an `i32` type value into the corresponding `i64` type value.
 
 ```k
-    syntax ConvOp ::= "extend_i32_u" | "extend_i32_s"
- // -------------------------------------------------
-    rule <k> i64 . extend_i32_u I => < i64 > I                               ... </k>
-    rule <k> i64 . extend_i32_s I => < i64 > #unsigned(i64, #signed(i32, I)) ... </k>
+    syntax CvtOp ::= "extend_i32_u" | "extend_i32_s"
+ // ------------------------------------------------
+    rule <k> i64 . extend_i32_u I:Int => < i64 > I                               ... </k>
+    rule <k> i64 . extend_i32_s I:Int => < i64 > #unsigned(i64, #signed(i32, I)) ... </k>
 
     rule #convSourceType(extend_i32_u) => i32
     rule #convSourceType(extend_i32_s) => i32
 ```
 
+Conversion turns a `int` type value to the corresponding `float` type value.
+
+```k
+    syntax CvtOp ::= "convert_i32_s" | "convert_i32_u" | "convert_i64_s" | "convert_i64_u"
+ // --------------------------------------------------------------------------------------
+    rule <k> FTYPE:FValType . convert_i32_s I:Int => #round( FTYPE , #signed(i32, I) ) ... </k>
```

Comment:
I'm not sure you need the explicit typing here, either, because `#round` only takes `FValType`.